shell/src/main/java/alluxio/cli/fsadmin/FileSystemAdminShellUtils.java

```diff
@@ -37,6 +47,34 @@ public static int compareTierNames(String a, String b) {
     return aValue - bValue;
   }
 
+  /**
+   * Checks if the master client service is running.
+   * Throws an exception if fails to determine that the master client service is running.
+   */
+  public static void masterClientServiceIsRunning() throws IOException {
```

Comment:
I would call this method `checkMasterClientServiceIsRunning`

Reply:
Changed, thanks